tox.ini

```
@@ -27,5 +27,6 @@ commands=
 [testenv:docs]
 description = Build documentation
 deps = -r{toxinidir}/docs/requirements.txt
+skip_install = True
 commands =
-    sphinx-build -E -W -d docs/build/doctrees -b html docs docs/build/html
+    sphinx-build -T -E -W --keep-going {tty:--color} -d docs/build/doctrees -b html docs docs/build/html
```

`--keep-going` alone doesn't do anything unless you also add a nitpicky mode.
You can do
```suggestion
    sphinx-build -T -E -n -W --keep-going {tty:--color} -d docs/build/doctrees -b html docs docs/build/html
```
to match what you've enabled on RTD. Or alternatively, add `nitpicky = True` to `docs/conf.py` and it'll do the same.

---

Another option that may be useful to add is `-j auto` unless you have extensions that don't support it.

---

The build is actually pretty quick right now. I don't know if we would benefit from parallelizing it.

---

It's just what people typically add by default so I thought I'd mention this.

---

I had forgotten about this lame note:

> For security reasons, parallel mode is disabled on macOS and python3.8 and above

---

Ah right, that corner case is probably annoying for people who tolerate macs :)

---

I fixed `sphinx` to use the proper start method. Doesn't seem to vastly improve the build time.

Single:
```
Executed in    5.50 secs    fish           external
   usr time    4.92 secs  107.00 micros    4.92 secs
   sys time    0.47 secs  957.00 micros    0.47 secs
```

Parallel:
```
Executed in    5.08 secs    fish           external
   usr time    7.63 secs    0.11 millis    7.63 secs
   sys time    1.45 secs    1.00 millis    1.45 secs
```

---

I went on a side quest. https://github.com/sphinx-doc/sphinx/pull/9793